Tabloid/client/src/components/PostForm.js

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from "react";
+import { useHistory } from "react-router-dom";
+import { Button, Container, Form, FormGroup, Input, Label } from "reactstrap";
+import { getAllCategories } from "../modules/categoryManager.js";
+import { addPost } from "../modules/postManager.js";
+
+export const PostForm = () => {
+    const [title, setTitle] = useState("");
+    const [content, setContent] = useState("");
+    const [header, setHeader] = useState(null);
+    const [categories, setCategories] = useState([]);
+    const [chosenCategory, setChosenCategory] = useState(null);
+    const [publicationDate, setPublicationDate] = useState(null);
+    const history = useHistory();
+
+    const getCategories = () => {
+        getAllCategories().then((cats) => setCategories(cats));
+    };
+
+    const makePost = (e) => {
+        e.preventDefault();
+        const post = {
+            title,
+            content,
+            ImageLocation: header,
```

Comment:
Watch your casing here.. Client side should all be camel case (when working in React especially).